staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -261,6 +261,10 @@ type VirtualMachineInstanceStatus struct {
 	// RuntimeUser is used to determine what user will be used in launcher
 	// +optional
 	RuntimeUser uint64 `json:"runtimeUser"`
+
+	// SELinuxContext is the actual SELinux context of the virt-launcher pod
+	// +optional
+	SelinuxContext string `json:"selinuxContext"`
```

Comment:
What about changing this into:
```
SelinuxContext *string `json:"selinuxContext,omitempty"`
```

so that on non-SELinux deployments this field is not shown at all?